megamek/unittests/megamek/client/bot/princess/NewtonAerospacePathRankerTest.java

```diff
@@ -29,10 +27,6 @@
 @RunWith(JUnit4.class)
 public class NewtonAerospacePathRankerTest {
 
-    private Princess mockPrincess;
-    private NewtonianAerospacePathRanker mockPathRanker;
-    private IGame mockGame;
-    private GameOptions mockGameOptions;
```

Comment:
Given how the below is... we should leave these here and ignore all of the tests.

Reply:
Added @SupressWarnings to leave the variables in tact.  The commented out test has the @ignore JUnit annotation.